components/tag-list/tag-list.js

```diff
@@ -1,8 +1,8 @@
 import { css, html, LitElement } from 'lit-element/lit-element.js';
 import { ArrowKeysMixin } from '../../mixins/arrow-keys-mixin.js';
-import { ifDefined } from 'lit-html/directives/if-defined.js';
 
-const ITEM_SPACING = 6;
+//const ITEM_SPACING = '6';
+const CSS_ITEM_SPACING = css`6`;
```

Comment:
Nit: 
```suggestion
const CSS_ITEM_SPACING = css`6`;
```

Reply:
Even though [Lit docs](https://lit.dev/docs/components/styles/#expressions) say to do this, [our linter hates it lol](https://github.com/BrightspaceUI/core/runs/5606443294?check_suite_focus=true#step:7:10).  Starting to think this isn't worth it lol, and if we ever want to change the 6 we can change the variable and the 4 places in css 😒